[2024-09-20T23:20:13.218+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-20T23:20:13.288+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag_v45.insert_inventory_data manual__2024-09-20T23:11:48.499967+00:00 [queued]>
[2024-09-20T23:20:13.337+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag_v45.insert_inventory_data manual__2024-09-20T23:11:48.499967+00:00 [queued]>
[2024-09-20T23:20:13.339+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-20T23:20:13.482+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): insert_inventory_data> on 2024-09-20 23:11:48.499967+00:00
[2024-09-20T23:20:13.503+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag_v45', 'insert_inventory_data', 'manual__2024-09-20T23:11:48.499967+00:00', '--job-id', '1631', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp58m5sj9s']
[2024-09-20T23:20:13.506+0000] {standard_task_runner.py:105} INFO - Job 1631: Subtask insert_inventory_data
[2024-09-20T23:20:13.514+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6149) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-20T23:20:13.517+0000] {standard_task_runner.py:72} INFO - Started process 6154 to run task
[2024-09-20T23:20:13.669+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag_v45.insert_inventory_data manual__2024-09-20T23:11:48.499967+00:00 [running]> on host 217a74bc1b7d
[2024-09-20T23:20:13.909+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dag_v45' AIRFLOW_CTX_TASK_ID='insert_inventory_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T23:11:48.499967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-20T23:11:48.499967+00:00'
[2024-09-20T23:20:13.911+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-20T23:20:13.936+0000] {sql.py:266} INFO - Executing: 
            TRUNCATE TABLE inventory;
            COPY inventory(id, name, stock_quantity, price)
            FROM '/shared/inventory.csv'
            DELIMITER ',' CSV HEADER;
        
[2024-09-20T23:20:13.968+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-20T23:20:14.142+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-20T23:20:14.187+0000] {sql.py:509} INFO - Running statement: 
            TRUNCATE TABLE inventory;
            COPY inventory(id, name, stock_quantity, price)
            FROM '/shared/inventory.csv'
            DELIMITER ',' CSV HEADER;
        , parameters: None
[2024-09-20T23:20:14.220+0000] {sql.py:518} INFO - Rows affected: 500
[2024-09-20T23:20:14.261+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-20T23:20:14.262+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag_v45, task_id=insert_inventory_data, run_id=manual__2024-09-20T23:11:48.499967+00:00, execution_date=20240920T231148, start_date=20240920T232013, end_date=20240920T232014
[2024-09-20T23:20:14.380+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-20T23:20:14.415+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-20T23:20:14.419+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
